### PR TITLE
Adding slackID variable for standalone mode

### PR DESCRIPTION
**Pull request by TehreemNisa**

### CODE REVIEW

File: bundle/manifests/grafanacloud.stakater.com_grafanacloudoperators.yaml [hunk 1]
```diff
@@ -76,6 +76,10 @@ spec:
                 - hubAndSpoke
                 - standalone
                 type: string
+              slackId:
+                description: Slack channel ID for standalone configuration , Grafana
+                  on Call app must be invited for this to work if the channel is private
```
Comment (bnallapeta) [inline bundle/manifests/grafanacloud.stakater.com_grafanacloudoperators.yaml:81]: typo: Grafana OnCall

**Review by bnallapeta — DISMISSED** [review]
Looks good. Minor typo.

File: bundle/manifests/grafanacloud.stakater.com_grafanacloudoperators.yaml (hunk 1 above)
Comment (TehreemNisa) [reply bundle/manifests/grafanacloud.stakater.com_grafanacloudoperators.yaml]: fixed

File: roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml [hunk 2]
```diff
@@ -45,8 +45,29 @@
     body:
       type: "alertmanager"
       name: "{{ cluster_name }}"
+      default_route:
+        slack:
+          channel_id: "{{ slack_id }}"
+          enabled: true
```
Comment (bnallapeta) [inline roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml:51]: where is this enabled: true coming from? is this part of the request to be sent to Grafana OnCall? Or is it part of our CR? could you explain this a bit?

File: roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml [hunk 3]
```diff
@@ -45,8 +45,29 @@
...
     status_code: [200, 201]
-  when: not integration_exists_for_cluster
+  when: not integration_exists_for_cluster and slack_id is defined
+  register: grafana_integration_response
+  failed_when: false
```
Comment (bnallapeta) [inline roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml:55]: setting failed_when to false basically passes the task even when the API request fails. Is this how we want it to be?

Comment (TehreemNisa) [reply roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml:55]: yes when it fails we should get an error , updating

File: roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml (hunk 2 above)
Comment (TehreemNisa) [reply roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml:51]: yes it is required for the body to be posted towards grafana on call when slackId is added, this is the format example

curl "https://oncall-prod-eu-west-0.grafana.net/oncall/api/v1/integrations/"   --request POST   --header "Authorization:"   --header "Content-Type: application/json"   --data '{
      "type":"alertmanager",
      "name":"tno4-ams-test",
      "default_route": {
        "slack": {
          "channel_id": "C06G60ZD4JZ",
          "enabled": true
        }
    }
}'

**Review by bnallapeta — APPROVED** [review]

**Pull request merged**